tests/unit/models/journal-test.js

```diff
@@ -0,0 +1,22 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Model | journal', (hooks) => {
+  setupTest(hooks);
+
+  test('it exists', function (assert) {
+    const store = this.owner.lookup('service:store');
+    const journal = store.createRecord('journal');
+    assert.ok(journal);
+  });
+
+  test('it computes pmcParticipation correctly', function (assert) {
```

Comment:
this might be the only test that was missing entirely that addresses that regresssion